extensions/wikia/Recirculation/templates/RecirculationController_Footer.php

```diff
@@ -0,0 +1,101 @@
+<div id="mixed-content-footer">
+	<!-- todo move this to proper place -->
```

Comment:
cards will be in mustache templates and rendered by JS. this will be done while working on fetching data